ext/intl/calendar/calendar_methods.cpp

```diff
@@ -347,11 +347,9 @@ static void _php_intlcal_before_after(
 	CALENDAR_METHOD_FETCH_OBJECT;
 
 	when_co = Z_INTL_CALENDAR_P(when_object);
-	/* Can this ever happen ? */
 	if (when_co->ucal == NULL) {
-		intl_errors_set(&co->err, U_ILLEGAL_ARGUMENT_ERROR,
-			"intlcal_before/after: Other IntlCalendar was unconstructed", 0);
-		RETURN_FALSE;
+		zend_argument_error(NULL, 2, "object is not fully initialized");
```

Comment:
I'm not sure the error message this generates will make sense.

Reply:
Is this better?
```suggestion
		zend_argument_error(NULL, 2, "must be constructed");
```

Reply:
Or maybe?
```suggestion
		zend_argument_error(NULL, 2, "is uninitialized");
```

Reply:
That's better :D
Will go with that and merge it directly then. :)